doc/source/whatsnew/v0.20.0.txt

```diff
@@ -705,6 +706,69 @@ New Behavior:
    TypeError: Cannot compare 2014-01-01 00:00:00 of
    type <class 'pandas.tslib.Timestamp'> to string column
 
+.. _whatsnew_0200.api_breaking.index_order:
+
+Index order after inner join due to Index intersection
```

Comment:
Make the title about what you have in the first sentence

Index.intersectio now preserves the order of the left Index